governance/votes/Executive vote - July 11, 2024.md

```diff
@@ -0,0 +1,125 @@
+summary: Transfer Dai to Aave as part of the SparkLend revenue sharing agreement, update the Chainlink Keeper Network Treasury Address, make Stability Fee and DSR changes as per the Stability Scope, and trigger a Spark Proxy Spell.
+date: 2024-07-11T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Aave Revenue Share, Update Keeper Network Address, Stability Scope Parameter Changes, Delegate Compensation, Spark Proxy Spell - July 11, 2024
+
+The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- **219,125 DAI** will be transferred to Aave as part of the Aave-SparkLend Revenue Sharing Agreement.
+- The Chainlink Keeper Network Treasury Address will be updated.
+- Multiple Stability Fees and the DSR will be updated.
+- **209.17 MKR** will be distributed to qualified Aligned Delegates.
+- A Spark Proxy Spell will be triggered at [0x91824fa4fd51E8440a122ffDd49C701F5C56D58e](https://etherscan.io/address/0x91824fa4fd51E8440a122ffDd49C701F5C56D58e).
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Aave-SparkLend Revenue Share Payment
+
+- **Authorization**: [Support Scope 9.4.1](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-4-q2-2024/24572)
+
+If this executive proposal passes, a payment of **219,125 DAI** will be made to a smart contract controlled by Aave Governance at [0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) as part of the Aave-SparkLend Revenue Share.
```

Comment:
I'd suggest adding that the payment is corresponding to Q2 2024

Reply:
Since it is detailed within the forum post, I'm going to leave this one out. Reasonable suggestion however, thanks for the careful review.

Reply:
Wording was deliberately chosen to be consistent with previous distributions of this nature.

e.g. https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20April%2022%2C%202024.md#aave-revenue-share-payment